### DISCUSSION TITLE
problem with css

### DISCUSSION THREAD

**Discussion by massimop1973** [gotenberg/gotenberg#487] [Q&A] [answered]
Hi,

I installed gotenberg using kubernetes on a local server, these are the configurations

[digital-ocean-ingress.yaml.txt](https://github.com/gotenberg/gotenberg/files/9443400/digital-ocean-ingress.yaml.txt)
[gotenberg-deployment.yml.txt](https://github.com/gotenberg/gotenberg/files/9443402/gotenberg-deployment.yml.txt)
[gotenberg-service.yaml.txt](https://github.com/gotenberg/gotenberg/files/9443403/gotenberg-service.yaml.txt)

making this call

```
curl \
--request POST 'htt.........' \
--header 'Content-Type: multipart/form-data' \
--form url='https://it.wikipedia.org/wiki/Pagina_principale' \
--form emulatedMediaType='screen' \
--form marginTop=0 \
--form marginBottom=0 \
--form marginLeft=0 \
--form marginRight=0 \
--form scale='0.8' \
-o wikipedia.pdf
```

I get a pdf file of the wikipedia page but the CSS looks wrong, this is the result:

[wikipedia.pdf](https://github.com/gotenberg/gotenberg/files/9443409/wikipedia.pdf)

using the demo installation, I get the correct one

```
curl \
--request POST 'https://demo.gotenberg.dev/forms/chromium/convert/url' \
--header 'Content-Type: multipart/form-data' \
--form url='https://it.wikipedia.org/wiki/Pagina_principale' \
--form marginTop=0 \
--form marginBottom=0 \
--form marginLeft=0 \
--form marginRight=0 \
--form scale='0.8' \
--form emulatedMediaType='screen' \
-o wikipedia3.pdf
```
[wikipedia3.pdf](https://github.com/gotenberg/gotenberg/files/9443420/wikipedia3.pdf)

thanks for your help

Massimo

**Comment by gulien** [comment, marked as answer]
Hello @massimop1973,

First, environment variables are not used anymore, so you should remove them ^^

Regarding your CSS issue, what's the exact version of Gotenberg you're using?

**Reply by massimop1973** [reply]
Thanks for your reply,

environment variables have been removed and still does not work

I'm using the version 7.3.0

**Reply by gulien** [reply]
The latest version is `7.5.4`, which is the version used in the demo 😄

**Reply by massimop1973** [reply]
I updated to the latest version and it works, thanks
